lib/engine/config/game/g_1830.rb

```diff
@@ -343,33 +343,33 @@ module G1830
         100
       ],
       "coordinates": "A19",
-      "color": "#d81e3e"
+      "color": "red"
     },
     {
       "float_percent": 60,
       "sym": "B&O",
       "name": "Baltimore & Ohio Railroad",
-      "logo": "1830/BO",
+      "logo": "1830/B&O",
       "tokens": [
         0,
         40,
         100
       ],
       "coordinates": "I15",
-      "color": "#0189d1"
+      "color": "blue"
     },
     {
       "float_percent": 60,
       "sym": "C&O",
       "name": "Chesapeake & Ohio Railroad",
-      "logo": "1830/CO",
+      "logo": "1830/C&O",
```

Comment:
we can''t use & in logos anymore, i think nginx strips them out or something

Reply:
Done.